### PR TITLE
Update PHP 7.0 to RC4

### PR DESCRIPTION
**Pull request by andrerom**
As soon as this is merged: https://github.com/docker-library/php/pull/139

### CODE REVIEW

**Comment by andrerom** [comment]
force pushed to re trigger tests /cc @yosifkit

File: library/php [hunk 1]
```diff
@@ -41,17 +41,17 @@ apache: git://github.com/docker-library/php@cf1e938f3721632443e01734bcfcbcf1160e
...
+7-cli: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7.0.0RC3: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7.0: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+
+7.0.0RC3-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+7.0-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+7-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+
+7.0.0RC3-fpm: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/fpm
```
Comment (yosifkit) [inline library/php]: It still has RC3 :open_mouth:, but the commit id is correct. If you have the php repo checked out, just run the `generate-stackbrew-library.sh` and direct the ouput to this file :+1:.

**Comment by andrerom** [comment]
fixed :blush:

File: library/php (hunk 1 above)
Comment (andrerom) [inline library/php]: fixed :relaxed:

**Comment by yosifkit** [comment]
LGTM, Build test of #1089; 317b297a39c127c351493ceaf1cde73af2cb1084 (`php`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/317b297a39c127c351493ceaf1cde73af2cb1084/library/php"
$ bashbrew build "$url"
Processing php:5.4.45-cli ...
Processing php:5.4-cli ...
Processing php:5.4.45 ...
Processing php:5.4 ...
Processing php:5.4.45-apache ...
Processing php:5.4-apache ...
Processing php:5.4.45-fpm ...
Processing php:5.4-fpm ...
Processing php:5.5.29-cli ...
Processing php:5.5-cli ...
Processing php:5.5.29 ...
Processing php:5.5 ...
Processing php:5.5.29-apache ...
Processing php:5.5-apache ...
Processing php:5.5.29-fpm ...
Processing php:5.5-fpm ...
Processing php:5.6.13-cli ...
Processing php:5.6-cli ...
Processing php:5-cli ...
Processing php:cli ...
Processing php:5.6.13 ...
Processing php:5.6 ...
Processing php:5 ...
Processing php:latest ...
Processing php:5.6.13-apache ...
Processing php:5.6-apache ...
Processing php:5-apache ...
Processing php:apache ...
Processing php:5.6.13-fpm ...
Processing php:5.6-fpm ...
Processing php:5-fpm ...
Processing php:fpm ...
Processing php:7.0.0RC4-cli ...
Processing php:7.0-cli ...
Processing php:7-cli ...
Processing php:7.0.0RC4 ...
Processing php:7.0 ...
Processing php:7 ...
Processing php:7.0.0RC4-apache ...
Processing php:7.0-apache ...
Processing php:7-apache ...
Processing php:7.0.0RC4-fpm ...
Processing php:7.0-fpm ...
Processing php:7-fpm ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing php:5.4.45-cli
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.4.45-apache
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.4.45-fpm
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'php-ext-install' [5/7]...passed
	'php-hello-world' [6/7]...passed
	'php-fpm-hello-web' [7/7]...passed
testing php:5.5.29-cli
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.5.29-apache
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.5.29-fpm
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'php-ext-install' [5/7]...passed
	'php-hello-world' [6/7]...passed
	'php-fpm-hello-web' [7/7]...passed
testing php:5.6.13-cli
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.6.13-apache
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:5.6.13-fpm
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'php-ext-install' [5/7]...passed
	'php-hello-world' [6/7]...passed
	'php-fpm-hello-web' [7/7]...passed
testing php:7.0.0RC4-cli
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:7.0.0RC4-apache
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'php-ext-install' [5/6]...passed
	'php-hello-world' [6/6]...passed
testing php:7.0.0RC4-fpm
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'php-ext-install' [5/7]...passed
	'php-hello-world' [6/7]...passed
	'php-fpm-hello-web' [7/7]...passed
```

**Comment by yosifkit** [comment]
Change is just a version bump, merging.

**Pull request merged**